alibi_detect/models/losses.py

```diff
@@ -199,3 +199,49 @@ def loss_adv_ae(x_true: tf.Tensor,
         return loss
     else:
         return loss
+
+
+def loss_distillation(x_true: tf.Tensor,
+                      y_pred: tf.Tensor,
+                      model: tf.keras.Model = None,
+                      loss_type: str = 'kld',
+                      temperature: float = 1.,
+                      ) -> tf.Tensor:
+    """
+    Loss function used for AdversarialAE.
+
+    Parameters
+    ----------
+    x_true
+        Batch of data points.
+    y_pred
+        Batch of prediction from the distilled model.
+    model
+        Keras model.
```

Comment:
`tf.keras model`. Just don't want to confuse people that only Keras can be used.